docs/docs/propeller/configuring.md

```diff
@@ -0,0 +1,50 @@
+# Configuring
+
+## Controlling throughput 
+
+Propeller has three parameters that work together to control throughput,
+`batchSize`, `maxConcurrentStreams` and `maxReadAhead`. The first controls the
+size of batches read from the source, the second how many streams can be
+concurrently processed, and the third how many batches can be buffered in
+memory.
+
+It is generally good to set `maxReadAhead` to at least `2` as that minimises
+downtime by allowing the feed to fetch a batch concurrently with stream
```

Comment:
It is generally useful to set `maxReadAhead` to at least `2` to minimize
idle time by allowing the feed to fetch the next batch during the handling of the streams. The number of events held in memory will be up to `batchSize *
maxReadAhead` (the last batch may be incomplete).

Reply:
addressed in #37